subprocess.hpp

```diff
@@ -1137,7 +1137,7 @@ inline void Popen::kill(int sig_num)
 }
 
 
-inline void Popen::execute_process() throw (CalledProcessError, OSError)
+inline void Popen::execute_process() noexcept
```

Comment:
Shouldn't this be noexcept(false) ? I still want to throw exception from this function, so it is definitely not "noexcept" which defaults to "noexcept(true)".

Reply:
You are right.  I didn't realize `noexcept` had such a parameter.  I have an update for this.